test/models/episode_story_handler_test.rb

```diff
@@ -69,15 +69,23 @@
       episode = EpisodeStoryHandler.create_from_story!(invalid_identifiers_story)
       episode.season_number.must_be_nil
       episode.episode_number.must_be_nil
-
     end
 
-   it 'allows identifiers to be zero' do
+   it 'does not allow identifiers to be zero' do
      podcast = create(:podcast, prx_uri: '/api/v1/series/32164')
      episode = EpisodeStoryHandler.create_from_story!(zero_identifiers_story)
-     episode.season_number.must_equal 0
-     episode.episode_number.must_equal 0
+     episode.season_number.must_be_nil
+     episode.episode_number.must_be_nil
    end
-  end
 
+   it 'blanks out identifiers on update' do
```

Comment:
This is the case that was failing - there was a value, then an update came though that should have blanked it out.